leaflet/static/leaflet/leaflet.css

```diff
@@ -633,8 +633,3 @@
 	margin-left: -12px;
 	border-right-color: #fff;
 	}
-
-/* Resize the "display_raw" textbox */
-.django-leaflet-raw-textarea {
-	width: 100%;

```

Comment:
Why did you also remove that part?

Reply:
Yes, that was removed as well. 
Why you do not want to try circle markers? It looks much better